sqlite3/lib/src/ffi/sqlite3.g.dart

```diff
@@ -342,6 +342,42 @@ class Bindings {
   late final _sqlite3_bind_parameter_count = _sqlite3_bind_parameter_countPtr
       .asFunction<int Function(ffi.Pointer<sqlite3_stmt>)>();
 
+  ffi.Pointer<sqlite3_char> sqlite3_bind_parameter_name(
```

Comment:
This file is auto-generated, please don't change it manually. Instead, add

```C
const char *sqlite3_bind_parameter_name(sqlite3_stmt*, int);
```

to `assets/sqlite3.h` and then use `dart run ffigen` to re-generate this file.